clients/rancher/client.go

```diff
@@ -113,6 +115,17 @@ func NewClient(bearerToken string, session *session.Session) (*Client, error) {
 
 	c.WranglerContext = wranglerContext
 
+	splitBearerKey := strings.Split(bearerToken, ":")
+	token, err := c.Management.Token.ByID(splitBearerKey[0])
+	if err != nil {
+		return nil, err
+	}
+
+	c.User, err = c.Management.User.ByID(token.UserID)
```

Comment:
I don't think we should get the user? I think we should just have the user ID as the client attribute and if any call needs anything extra it can get the user. That way we limit our instantiation to one GET call.

Reply:
Ok, I'll make the change.